src/app/app.component.html

```diff
@@ -11,7 +11,9 @@
       <ion-toolbar color="primary">
         <ion-title>{{ sideMenuDefaults.title }}</ion-title>
         <div class="app-version" slot="end">
-          <span *ngIf="sideMenuDefaults.should_show_version">{{ APP_VERSION }} </span>
+          <span *ngIf="sideMenuDefaults.should_show_version"
+            >{{ APP_VERSION }}{{ CONTENT_VERSION ? "(" + CONTENT_VERSION + ")" : "" }}
```

Comment:
nit(blocking)
I'm guessing content_version is more interesting to the user anyway? How about we add a span here with an ngIf statement so that if there is content version we just show the content version and use a [title attribute](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_global_title) to display the full title with app_version?

A second span could just show regular app version if no content version exists, or possibly some other text to make it clear that there is not versioned content (would there be a way to ensure we do have versioned content?)

Reply:
Thanks, I've implemented this. As the title attribute content doesn't display on touch screens (no hover functionality), I've custom CSS to display the content on click when on mobile, see [latest commit](https://github.com/IDEMSInternational/parenting-app-ui/pull/2111/commits/bf2b55a923c4547f0dd599b383cc1f4a362a8be9).